docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,98 @@
+
+### Fast development, less flexibility
+
+Most modern internal tool builders in the market fall into this category. Products like Retool, Appsmith act like a SaaS and provide a UI for rapid application development. The apps can be either hosted by you or them. They are excellent products for someone who's looking to build fast. The caveats are that you can't use your IDE, your existing codebase, or logic, and it can cost a lot as you scale.
+
+### Moderate development speed, moderate flexibility
+
+Products like airplane.dev, interval.com fall under this category. They have APIs for all admin use cases; while this can be great for those who enjoy writing code, it is a bit too much as adding a component also requires writing code. Their main advantage over previous category is that they store their configuration in local code files. This puts more control over the development lifecycle back in the developer's hands. Users can version control, deploy, and use any IDE they want.
+
+### Slow development speed, more flexibility
+
+Products like react admin, refine.dev fall in this category, they are all code and have APIs for everything to build an admin panel from scratch. They don't support cloud hosting and are a relatively cheaper alternative.
+
+We'll do a detailed comparison with these tools in the future.
+
+## Toolpad
+
+How is Toolpad different from the above tools, and why should you use it? Toolpad is a **flexible and fast** way to build a back office application. Let's see how:
+
+### Front end UI
+
+Toolpad works as a drag-and-drop UI builder and uses MUI's own Material UI components. Material UI has almost every component required for back-office use cases; currently, 16 of them are there in Toolpad but we are always adding more.
+
+For more components, Toolpad supports adding any number of custom components. You can create your own by writing React code.
+
+Drag-and-drop builder helps you assemble a quick UI and gives you a head start for the next steps.
+
+### Consistent Design
+
+Maintaining a consistent design language throughout a web application is crucial for a seamless user experience. Toolpad helps you achieve this by providing pre-built components that follow Material Design guidelines. This ensures your application looks consistent across all pages and screens.
+
+Toolpad also supports adding a theme to a page, it can help with a UI that matches your brand guidelines. The components have overrides in case wish to customize them further.
```

Comment:
```suggestion
Toolpad also supports adding a theme to a page which can help with a UI that matches your brand guidelines. The components have individual overrides as well, in case you wish to customize them further.
```